libbeat/cmd/instance/metrics_common.go

```diff
@@ -35,7 +36,11 @@ func init() {
 	beatMetrics = monitoring.Default.NewRegistry("beat")
 	monitoring.NewFunc(beatMetrics, "info", reportInfo, monitoring.Report)
 
-	ephemeralID = uuid.NewV4()
+	var err error
+	ephemeralID, err = uuid.NewV4()
```

Comment:
What's the impact to monitoring if the ephemeral ID is Nil?

Reply:
The beat won't show up in some visualizations.